CHANGELOG.md

```diff
@@ -5,10 +5,15 @@ All notable changes in io-sdk-golang will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
 ## [0.3.0] - 2023-08-21
 
 - Modified query options encoding for optimal use with CCO microservices
 
+## [0.2.2] - 2023-10-02
```

Comment:
Looks like this one needs to be above the above one and with a later version?

Reply:
omfg... there is a v0.3.0, but the Makefile wasn't updated. I think this is my bad.